# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vx32/hotfix-signal.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- src/9vx/kprocdev.c.vanilla	2025-10-18 14:18:37.146547748 +0200
+++ src/9vx/kprocdev.c	2025-10-18 14:18:46.023547266 +0200
@@ -9,6 +9,7 @@
 
 #include "u.h"
 #include <pthread.h>
+#include <signal.h>
 #include "lib.h"
 #include "mem.h"
 #include "dat.h"
